README.md

```diff
@@ -69,7 +69,7 @@ Teleport source code consists of the actual Teleport daemon binary written in Go
 it has a web UI (located in /web directory) written in Javascript. The WebUI is not changed often
 and we keep it checked into Git under `/dist`, so you only need to build Golang:
 
-Make sure you have Golang `v1.12` or newer, then run:
+Make sure you have Golang `v1.13` or newer, then run:
```

Comment:
I'm not sure which release we pushed this with, but here are the relevant PRs.

Teleport Commit:
https://github.com/gravitational/teleport/pull/3109/

Internal Build tools: 
https://github.com/gravitational/ops/commit/fcb6b8ec2ee75fd2f9bdaabe211a1c569fe9a103#diff-f618ae0dbd43b7405aabce444dd93144 


Looking at our internal Slack we bumped GoLang to resolve this issue  `https://github.com/golang/go/issues/34540`, but we went straight to v.1.13.

Reply:
Based on the tag list https://github.com/gravitational/teleport/commit/e6a024b820556b1335c6f366af1677995f30b433 seen here:

```
walt@work:~/git/teleport$ git tag --contains e6a024b8                                       
v4.2.0
v4.2.0-alpha.3
v4.2.0-alpha.4
v4.2.0-alpha.5
v4.2.0-dev.1
v4.2.0-dev.2
v4.2.0-dev.3
v4.2.0-dev.4
v4.2.0-rc.1
v4.2.0-rc.2
v4.2.0-rc.3
v4.2.1-rc.1
```

It looks like 1.13 is a new req in 4.2.  I know we don't touch releases, so I propose:

We update the release notes in CHANGELOG to reflect this, and add a note to the next 4.2.X, which @fspmarshall is working on today.

Reply:
From a semver standpoint, should we be upgrade our go base on a minor 4.X release?  I'm not yet familiar enough with the go world to know if this would be considered a breaking change.

It seems like since we build everything into a single (statically linked?) binary, the end user shouldn't really care.  However, if anyone is integrating with teleport programmatically, (using the go API), they'd probably care about a 1.12 -> 1.13 requirement bump.

Reply:
@russjones @klizhentas What do you think?

The binary isn't quite statically linked as it has a depedency on a given libc version. Other than that though the Go runtime used shouldn't make any difference to someone - it's just if they're trying to build from source.